packages/wrangler/src/utils/compatibility-date.ts

```diff
@@ -3,6 +3,16 @@ import { configFileName } from "../config";
 import { logger } from "../logger";
 import type { Config } from "../config";
 
+/**
+ * Returns the compatibility date to use in development.
+ *
+ * When no compatibility date is configured, used the workerd date.
+ * That's the latest valid compatibility date.
```

Comment:
It's not strictly the latest valid date. It is the date of the currently installed Miniflare's workerd. There maybe a new workerd version published which this version of Wrangler/Miniflare doesn't yet depend upon.

Reply:
Do you have a suggestion on how to word this to be clearer?